jsHelper/spicetifyWrapper.js

```diff
@@ -111,8 +114,12 @@ window.Spicetify = {
 		skipForward: (amount = 15e3) => {
 			Spicetify.Player.origin.seekForward(amount);
 		},
+		setHeart: b => {
+			const uri = [Spicetify.Player.origin._state.item?.uri];
+			b ? Spicetify.Player.origin._collection.add({ uri }) : Spicetify.Player.origin._collection.remove({ uri });
```

Comment:
if you don't assign to the variable, don't use conditional ternary operator please
```suggestion
			if (b) Spicetify.Player.origin._collection.add({ uri })
			else Spicetify.Player.origin._collection.remove({ uri });
```

Reply:
you're just jealous of it coolness

Reply:
xd